router/docs/Profiling.md

```
@@ -1,68 +1,78 @@
+To run a solid workflow and get a sense of the routers performance, you can
+use [k6](https://grafana.com/docs/k6/latest/).
 To do so, run:
+
 ```shell
 brew install k6
 k6 run bench.js
 ```
 
 This will run a benchmark on the router, and you can see the results in the terminal.
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)